drivers/media/i2c/imx296.c

```diff
@@ -384,10 +384,35 @@ static const struct v4l2_ctrl_ops imx296_ctrl_ops = {
 	.s_ctrl = imx296_s_ctrl,
 };
 
+static void imx296_setup_hblank(struct imx296 *sensor, unsigned int width)
+{
+	/*
+	 * Horizontal blanking is controlled through the HMAX register, which
+	 * contains a line length in INCK clock units. The INCK frequency is
+	 * fixed to 74.25 MHz. The HMAX value is currently fixed to 1100,
```

Comment:
I believe this is correct.  The sensor internally multiplies any INCLK value to 74.25 MHz.  This is done by setting the INCLK value in one of the clk related registers. This can be verified by the frame time computations, which do give the right numbers, implying HMAX is at the right value.

Reply:
That would imply that the comment is misleading. Not your comment, so not a blocker for this PR though.
These patches really ought to be upstreamed though, hence me being pedantic.

Reply:
I could reword the comment as:

```
	 * Horizontal blanking is controlled through the HMAX register, which
	 * contains a line length in units of an internal 74.25 MH clock derived from the INCLK...
```

Reply:
> These patches really ought to be upstreamed though, hence me being pedantic.

Yes, no reason not to send these patches upstream.